src/components/guess-entry/guess-entry.css

```diff
@@ -3,7 +3,7 @@
   text-align: center;
 }
 
-.guess-entry:not(:last-child) {
+.guess-entry {
```

Comment:
Removed :not(:last-child) because we realized this wasn't actually doing anything.